var/spack/repos/builtin/packages/mvdtool/package.py

```diff
@@ -63,3 +65,11 @@ def cmake_args(self):
                 '-DBUILD_PYTHON_BINDINGS:BOOL=ON'
             ])
         return args
+
+    @when('+python')
+    def setup_dependent_environment(self, spack_env, run_env, dependent_spec):
+        site_dir = self.spec['python'].package.site_packages_dir.split(os.sep)[1:]
```

Comment:
In cases python is external, wont `spec['python']` fail?

Reply:
With external Python, `spec['python']` should still be present. Everything that Python depends on should be truncated.